src/EditCell.tsx

```diff
@@ -2,31 +2,25 @@ import React, { useState, useCallback } from 'react';
 import clsx from 'clsx';
 
 import EditorContainer from './editors/EditorContainer';
-import { CellRendererProps, SharedEditorContainerProps, SharedEditorProps } from './types';
+import { CellRendererProps, SharedEditorProps } from './types';
 
 type SharedCellRendererProps<R, SR> = Pick<CellRendererProps<R, SR>,
   | 'rowIdx'
   | 'row'
   | 'column'
 >;
 
-interface EditCellRendererProps<R, SR> extends SharedCellRendererProps<R, SR>, Omit<React.HTMLAttributes<HTMLDivElement>, 'style' | 'children'> {
-  editorPortalTarget: Element;
-  editorContainerProps: SharedEditorContainerProps;
+interface EditCellProps<R, SR> extends SharedCellRendererProps<R, SR>, Omit<React.HTMLAttributes<HTMLDivElement>, 'style' | 'children'> {
```

Comment:
We should use the `Omit` from our `types`. Should double-check other `Omit` usage.